frontend/src/i18n/locales/nl.ts

```diff
@@ -35,7 +35,7 @@ export default {
         no_submission_files: "Geen indieningen gevonden",
     },
     submission: {
-        status: "Indiening status: {status}",
+        status: "Indiening status:",
```

Comment:
```suggestion
        status: "Status indiening:",
```
Klinkt beter Nederlands